scripts/test/test_util.js

```diff
@@ -177,13 +177,14 @@ var CommandlineTest = function(command) {
             flushAndFail('expecting test max duration: ' +
                 self.expectedMaxTestDuration_ + ', actual: ' + duration);
       }
-    }).fin(function() {
```

Comment:
Perhaps you are thinking about Bluebird promises? `finally`, from all I can tell does not appear to be a method in the ES6 spec, as a static nor instance method.